requirements.txt

```
@@ -1,6 +1,6 @@
 pytest>=2.0.3
 pytest-cov>=1.8.1
-lxml>=2.3
+lxml==5.2.2
```

```suggestion
lxml>=4.0.0
```

As far as I can tell, lxml 4.0 support the XPath callable object instead of the `evaluate` method, and I don't see the necessity to force that specific version of lxml here.